### PR TITLE
Preparations for refactoring trigger rules

### PR DESCRIPTION
**Pull request by mik-laj**
Make sure you have checked _all_ steps below and that above description is correct.

### JIRA
No JIra. It's preparation to refactor trigger rule.

### Tests
- [ ] My PR has unit tests testing the functionality (or I explained why it needs no tests)

### Commits
- [ ] My commits reference the issue in the message using: Fixes #XXX /Closes #XXX or similar.

### Documentation
- [ ] My PR adds documentation that describes how to use it (or I explained why it needs no docs)

### CODE REVIEW

File: o2a/converter/oozie_converter.py [hunk 1]
```diff
@@ -111,6 +120,55 @@ def convert_nodes(nodes: Dict[str, ParsedActionNode]):
             p_node.tasks = tasks
             p_node.relations = relations
 
+    @staticmethod
+    def convert_dependencies(workflow: Workflow) -> None:
+        """
+        Create a python imports based on nodes.
```
Comment (potiuk) [inline o2a/converter/oozie_converter.py]:
```suggestion
        Create Python imports based on nodes.
```

File: o2a/converter/oozie_converter.py [hunk 2]
```diff
@@ -121,6 +128,47 @@ def convert_dependencies(workflow: Workflow) -> None:
         for node in workflow.nodes.values():
             workflow.dependencies.update(node.mapper.required_imports())
 
+    @staticmethod
+    def convert_relations(workflow: Workflow) -> None:
+        """
+        Given a dictionary of task_ids and ParsedActionNodes,
+        returns a set of logical connectives for each task in Airflow.
+
+        :return: Set with strings of task's downstream nodes.
```
Comment (potiuk) [inline o2a/converter/oozie_converter.py]: This comment and :Return: are wrong. The logical connections are stored in workflow.relations and not returned.

File: o2a/converter/oozie_converter.py [hunk 3]
```diff
@@ -121,6 +128,47 @@ def convert_dependencies(workflow: Workflow) -> None:
...
+                workflow.relations.add(relation)
+
+    @staticmethod
+    def update_trigger_rules(workflow: Workflow) -> None:
+        """
+        Updates the trigger rules of each node based on the downstream and
+        error nodes.
+        """
+        for node in workflow.nodes.values():
+            # If a task is referenced  by an "ok to=<task>", flip bit in parsed
```
Comment (potiuk) [inline o2a/converter/oozie_converter.py]: This comment is wrong. I think it should be removed as the code is clear enough that it does not need extra explanation.

Comment (mik-laj) [reply o2a/converter/oozie_converter.py]: I updated the PR. Thanks.

**Pull request merged**